content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -1,54 +1,91 @@
 ---
-title: Plugging in existing CA Certificates
-description: Shows how system administrators can configure Istio's CA with an existing root certificate, signing certificate and key.
+title: Plug in CA Certificates
+description: Shows how system administrators can configure Istio's CA with a root certificate, signing certificate and key.
 weight: 80
 keywords: [security,certificates]
 aliases:
     - /docs/tasks/security/plugin-ca-cert/
 test: true
```

Comment:
Test needs to be updated.

Reply:
@frankbu there is no test currently ... do you mean snips.sh? It's updated now.

Reply:
oh nm ... looks like it was just checked in.  I see it now ... will update.